src/TestResults/TestResultProvider.ts

```diff
@@ -39,141 +31,39 @@ export class TestResultProvider {
-            remainingAssertions.splice(aIndex, 1);
-          }
-          return;
-        }
-
-        let err: string;
-        if (_onMatchError) {
-          err = _onMatchError(test, matched);
-        }
-        // if there is an error string, create a test result with it
-        if (err) {
-          results.push(toMatchResult(test, undefined, err));
-          return;
-        }
-
-        if (_trackRemaining) {
-          remainingTests.push(test);
-        }
-      });
-      return [results, remainingTests, remainingAssertions];
-    };
-
     if (this.resultsByFilePath[filePath]) {
       return this.resultsByFilePath[filePath];
     }
-    const matchPos = (t: ItBlock, a: TestAssertionStatus): boolean =>
-      (a.line !== undefined && a.line >= t.start.line && a.line <= t.end.line) ||
-      (a.location && a.location.line >= t.start.line && a.location.line <= t.end.line);
-
-    const matchName = (t: ItBlock, a: TestAssertionStatus): boolean => t.name === a.title;
-    const templateLiteralPattern = /\${.*?}/; // template literal pattern
-    const matchTemplateLiteral = (t: ItBlock, a: TestAssertionStatus): boolean => {
-      if (!t.name.match(templateLiteralPattern)) {
-        return false;
-      }
-      const parts = t.name.split(templateLiteralPattern);
-      const r = parts.every((p) => a.title.includes(p));
-      return r;
-    };
-    const onMatchError: OnMatchError = (t: ItBlock, match: TestAssertionStatus[]) => {
-      let err: string;
-      if (match.length <= 0 && t.name.match(templateLiteralPattern)) {
-        err = 'no test result found, could be caused by template literals?';
-      }
-      if (match.length > 1) {
-        err =
-          'found multiple potential matches, could be caused by duplicate test names or template literals?';
-      }
-      if (err && this.verbose) {
-        // tslint:disable-next-line: no-console
-        console.log(`'${t.name}' failed to find test result: ${err}`);
-      }
-      return err;
-    };
 
-    let { itBlocks } = parseTest(filePath);
-    let assertions = this.reconciler.assertionsForTestFile(filePath) || [];
-    const totalResult: TestResult[] = [];
-
-    if (assertions.length > 0 && itBlocks.length > 0) {
-      const algorithms: Array<[IsMatched[], OnMatchError]> = [
-        [[matchName, matchPos], undefined],
-        [[matchTemplateLiteral, matchPos], undefined],
-        [[matchTemplateLiteral], undefined],
-        [[matchName], onMatchError],
-      ];
-      for (const [matchers, onError] of algorithms) {
-        let result: TestResult[];
-        [result, itBlocks, assertions] = matchTests(itBlocks, assertions, matchers, onError);
-        totalResult.push(...result);
-        if (itBlocks.length <= 0 || assertions.length <= 0) {
-          break;
-        }
-      }
-    }
+    let matchResult: TestResult[] = [];
 
-    // convert remaining itBlocks to unmatched result
-    itBlocks.forEach((t) => totalResult.push(toMatchResult(t)));
```

Comment:
not really, the unmatched assertions will be ignored because there is no way to report them. Keep in mind these match results are used to generate decorators for the test files, i.e. the itBlocks...